docker/Dockerfile

```diff
@@ -1,19 +1,50 @@
-FROM rstudio/r-base:4.3.2-jammy
-# does not contain rstudio, just r-base installation
-# is based on ubuntu:22.04
+# use a base image with Ubuntu
+FROM ubuntu:20.04
 
-# install inkscape
-RUN apt-get update -y \
-    && apt-get install -y --no-install-recommends \
-        inkscape \
-    && rm -rf /var/lib/apt/lists/*
-RUN mkdir -p /var/build
+# set environment variables to non-interactive to avoid prompts
+ENV DEBIAN_FRONTEND=noninteractive
+ENV R_HOME=/usr/lib/R
+ENV INKSCAPE_HOME=/usr/bin/inkscape
 
-COPY worldmap-svg.R /var/build/worldmap-svg.R
-COPY geocoding.py /var/build/geocoding.py
-COPY process.sh /var/build/process.sh
+# install required packages and dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    software-properties-common \
+    python3 \
+    python3-pip \
+    r-base \
+    && apt-get clean
 
+# create symlink for `python` to point to `python3`
+RUN ln -s /usr/bin/python3 /usr/bin/python
+ 
 
-# TODO add geopy python image
+# install Python packages
+RUN pip3 install --no-cache-dir \
+    geopy 
 
-CMD ["/bin/bash"]
+
+#install Inkscape
+RUN add-apt-repository ppa:inkscape.dev/stable -y && \
+    apt-get update && \
+    apt-get install -y inkscape
+     
+# install R packages
+RUN R -e "install.packages(c('maps', 'mapproj', 'mapdata'), repos='https://cloud.r-project.org/')"
+
+# set working directory
+WORKDIR /var/build
+
+# copy necessary files into the Docker image
+COPY collaborators-worldmap/scripts/worldmap-svg.R /var/build/worldmap-svg.R
+COPY collaborators-worldmap/scripts/geocoding.py /var/build/geocoding.py
+COPY collaborators-worldmap/scripts/process.sh /var/build/process.sh
+COPY collaborators-worldmap/target /var/build/target
+COPY reporting /var/build/reporting
```

Comment:
You are mixing the input data (`collections-metadata.tsv` and `coords.tsv`) with the scripts during building of the docker container.
E.g. when the input data changes, we will need to build a new docker container, which is bad.
Instead, the docker container should contain only the required programs (e.g. inkscape, R, python) in specific versions as well as the scrips required to build the map image. A new image should only be required, once the code or application changes and not when the input data changes.
The input data should be mounted into the docker container when running the docker container.
E.g. when the input data changes, no docker build should be needed.